### PR TITLE
Merging of subcommands

### LINKED ISSUE #96: About the merging of subcommands
I do not understand why the image-tools in the three sub-command split into three software, I think these three commands should be combined into a tool, after all, they belong to image-tools, I think the merger will Easy to understand and application, if I was a user, I would think that the use of a software implementation of the three commands than the use of three software to implement a command to be more convenient.
I currently have a plan to merge, don't know how you think .

### PR DESCRIPTION
**Pull request by q384566678**
This is some idea about #96 

Fixes #96 
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by vbatts** [comment]
i think conceptually this SGTM

**Comment by xiekeyang** [comment]
SGTM. need rebase.

**Comment by q384566678** [comment]
@vbatts @xiekeyang I think this change will be better. PTAL

**Comment by xiekeyang** [comment]
@q384566678 It seems that `log-level` is in progress on #46 . In that patch, @stevvooe suggest that we should not use `log-level`, instead to use debug mode. And you seems not replace fmt.Print by `logrus` with `context`. So I think you seems to had better to remove this stuff from this patch.

File: man/oci-image-tool.1.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+**--log-level**=LEVEL
+  Log level (panic, fatal, error, warn, info, or debug) (default: "error").
+
+**-v**, **--version**
+  Print version information.
+
+# COMMANDS
+**validate**
+  Validate one or more image files
```
Comment (stevvooe) [inline man/oci-image-tool.1.md]: What is an "image file"?

File: man/oci-image-tool.1.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  See **oci-image-tool-validate**(1) for full documentation on the **validate** command.
+
+**unpack**
+  Unpack an image or image source layout
```
Comment (stevvooe) [inline man/oci-image-tool.1.md]: What is an "image" or "image source layout"?

File: man/oci-image-tool.1.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  See **oci-image-tool-unpack**(1) for full documentation on the **unpack** command.
+
+**bundleCreate**
```
Comment (stevvooe) [inline man/oci-image-tool.1.md]: Let's avoid camelCase in our command designs. This should employ `<object> <verb>` or use dashes `bundle-create`.

File: cmd/oci-image-tool/unpack.go [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+	if err != nil {
+		fmt.Printf("unpacking failed: %v\n", err)
+	}
+	return err
+}
+
+var unpackCommand = cli.Command{
+	Name:   "unpack",
+	Usage:  "Unpack an image or image source layout",
+	Action: unpackHandle,
```
Comment (stevvooe) [inline cmd/oci-image-tool/unpack.go]: `unpackHandler`.

File: man/oci-image-tool.1.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+**-v**, **--version**
+  Print version information.
+
+# COMMANDS
+**validate**
+  Validate the given file(s) against the OCI image specification
+  See **oci-image-tool-validate**(1) for full documentation on the **validate** command.
+
+**unpack**
```
Comment (stevvooe) [inline man/oci-image-tool.1.md]: This language still doesn't make the different between "unpack" and "create" clear.

**Comment by stevvooe** [comment]
LGTM

Please follow up on the issues with `create` / `unpack`.

**Comment by vbatts** [comment]
LGTM

**Comment by xiekeyang** [comment]
@q384566678 @vbatts @stevvooe This PR call `logrus` function, without introduce it to `vendor`. It  leads build fail on `make tool`:
```
$ make tool 
go build -o oci-image-tool ./cmd/oci-image-tool
cmd/oci-image-tool/main.go:21:2: cannot find package "github.com/Sirupsen/logrus" in any of:
```
At least we should import `logrus` to `glide.yaml` and then `make update-deps`, the package can be generated in `vendor`.
Please take a review, is it Right?

**Pull request merged**